packages/coq-catt-plugin/coq-catt-plugin.1.0/opam

```diff
@@ -11,7 +11,7 @@ bug-reports: "https://github.com/thibautbenjamin/catt/issues"
 depends: [
   "dune" {>= "3.16"}
   "catt"
-  "coq" {>= "8.20"}
+  "coq" {>= "8.20" & < "9.0"}
```

Comment:
BTW, unrelated to this PR, this package should probably live in the Coq Opam repo ( https://github.com/coq/opam ) rather than here. Cc @thibautbenjamin